fmings/limeify-be
# Update playlist

## Description
Added category into playlist model.
Also changed the get single playlist call to include album, artist, and category.

## Related Issue
Front end Detail for playlist page

## Motivation and Context
We want the user to be able to see the list of songs on the playlist detail page with more information about that single song that was added. 

## How Can This Be Tested?

1. git fetch origin updatePlaylist
2. git checkout updatePlaylist
3. run in terminal: dotnet ef database update
4. when you run code you should have swagger open and then you can test call (GET /api/playlists/{id}) 
this should return all information added to the call.

## Screenshots (if appropriate):
![image](https://github.com/user-attachments/assets/ccdd93b1-76d9-483b-abeb-76f23ad462ec)

## Types of changes

- [x ] New feature (non-breaking change which adds functionality)
